packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -346,16 +346,16 @@ export const GRID_COLUMN_HEADER_DRAG_END = 'columnHeaderDragEnd';
 export const GRID_SELECTION_CHANGED = 'selectionChange';
 
 /**
- * Fired when the current page has changed. Called with a [[GridPageChangeParams]] object.
+ * Fired when the current page change. Called with a [[GridPageChangeParams]] object.
  * @event
  */
-export const GRID_PAGE_CHANGED = 'pageChange';
+export const GRID_PAGE_CHANGE = 'pageChange';
```

Comment:
Per our naming convention, this is both a breaking change and a bug fix, depending on how you see it. Since we are leaning toward documenting `pageChange` and have developers figure out the import name (based on @m4theushw PR), I would see this more as a bug fix.

Reply:
Yes, I would keep it as a breaking change, but once the documentation for events is up we could stop documenting constant **name** changes and only focus on constant **value** changes.